ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/openshift-image-based-upgrade-e2e-conf-commands.sh

```diff
@@ -15,13 +15,27 @@ remote_workdir=$(cat ${SHARED_DIR}/remote_workdir)
 instance_ip=$(cat ${SHARED_DIR}/public_address)
 host=$(cat ${SHARED_DIR}/ssh_user)
 ssh_host_ip="$host@$instance_ip"
-TARGET_VM_NAME=$(cat ${SHARED_DIR}/target_vm_name)
-target_kubeconfig=${remote_workdir}/ib-orchestrate-vm/bip-orchestrate-vm/workdir-${TARGET_VM_NAME}/auth/kubeconfig
+
+TEST_VM_NAME="unknown"
+case $TEST_CLUSTER in
+  "target")
+    TEST_VM_NAME=$(cat ${SHARED_DIR}/target_vm_name)
+    ;;
+  "seed")
+    TEST_VM_NAME=$(cat ${SHARED_DIR}/seed_vm_name)
+    ;;
+  *)
+    echo "Unknown image tag format specified ${SEED_IMAGE_TAG_FORMAT}"
```

Comment:
Shouldn't we print an error on the TEST_CLUSTER here instead of SEED_IMAGE_TAG_FORMAT?

Reply:
This got pulled out with some refactoring to an if statement :+1: